ouroboros-consensus-cardano-tools/src/Cardano/Node/Protocol/Cardano.hs

```diff
@@ -1,8 +1,10 @@
+{-# LANGUAGE DataKinds #-}
```

Comment:
No annoyance at all. I am really used to a CI job telling me when formatting is off. :smile: 

Fixed.

You guys should add a CI job that fails if stylish-haskell wasn't run on the file, this would avoid the extra work for the reviewer to pay attention to that stuff.